src/Error/SerializationError.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Error;
+
+use Exception;
+
+/**
+ * Thrown when failing to serialize a scalar value.
```

Comment:
https://github.com/webonyx/graphql-php/pull/903/files#r679674725

Ah, then this needs to be modified